john_pham/Assignments/OOP/products.py

```diff
@@ -0,0 +1,44 @@
+class Product(object):
```

Comment:
A+ on this assignment. Great naming conventions and coding structure/syntax with the added bonus of working as expected without errors. Great job!